src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -763,3 +767,444 @@ def test_redirect_from_other_pages(self):
         r = self.client.get(reverse('devhub.addons.edit', args=['a3615']),
                             follow=True)
         self.assert3xx(r, reverse('devhub.submit.details', args=['a3615']))
+
```

Comment:
new tests from here on.  The functionality that's been tested is similar to the tests for the old flow in test_views.py though and we'll be able to delete those once that code is deleted.